scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/CalendarInfoWindow.java

```diff
@@ -200,8 +200,14 @@ private void loadWindowsContent(RequestBuilder.Method method) {
 
         String host = com.google.gwt.user.client.Window.Location.getHostName();
         String user = LoginModel.getInstance().getLogin();
-        String requestUrl = "http://" + host + ":" + SchedulerConfig.get().getCalendarServerPort() +
-            "/calendar-service/private-urls/" + user + "/";
+        String requestUrl = null;
+
+        if ("http".equals(SchedulerConfig.get().getCalendarServerProtocol())) {
+            requestUrl = "http://" + host + ":" + SchedulerConfig.get().getCalendarServerPort() +
+                "/calendar-service/private-urls/" + user + "/";
+        } else {
+            requestUrl = "https://" + host + "/calendar-service/private-urls/" + user + "/";
```

Comment:
The following part could be put outside the if  : 

requestUrl  +=  "/calendar-service/private-urls/" + user + "/";